tslint.json

```
@@ -65,7 +65,7 @@
     "no-arg": true,
     "no-bitwise": false,
     "no-consecutive-blank-lines": [true],
-    "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
+    "no-console": false,
```

```suggestion
    "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
```

在使用的地方 tslint:disable 就行, 否则 lint 无法发现其他地方是否误用